llpc/util/llpcElfWriter.h

```diff
@@ -74,6 +74,22 @@ template <class Elf> class ElfWriter {
 
   static void updateMetaNote(Context *context, const ElfNote *note, ElfNote *newNote);
 
+  static size_t getRelocsCount(const SectionBuffer *pRelocSection);
+
+  static size_t getRelocPsStartPos(const SectionBuffer *pRelocSection, size_t psIsaOffset);
+
+  void createNewSection(const char *pSectionName, SectionBuffer *pInputSection, SectionBuffer *pNewSection);
+
+  void mergeRelocSection(const ElfReader<Elf> &reader, const SectionBuffer *pSection1, size_t section1RelocsCount,
+                         const SectionBuffer *pSection2, size_t section2RelocsOffset, int64_t deltaOfPsOffset,
+                         SectionBuffer *pNewSection);
+
+  void addRelocSymbols(const ElfReader<Elf> &reader, typename Elf::Reloc inputRelocs);
```

Comment:
Maybe add a typedef for `Elf::Reloc` just like it's done for `SectionBuffer`?

Reply:
Addressed.